docs/contributor-guide/beta-testing.md

```diff
@@ -0,0 +1,106 @@
+<!-- Source: https://github.com/arduino/tooling-project-assets/blob/main/documentation-templates/contributor-guide/application/contributor-guide/beta-testing.md -->
+
+# Beta Testing Guide
+
+Beta testing of development versions is a valuable contribution to the project. You can help to ensure the quality of the production release that will be distributed to the user community.
+
+Builds of the project are automatically created after every relevant change to the project in order to make it easy for anyone to participate in the testing effort.
+
+---
+
+❗ Make sure to always download the newest available tester build in order to ensure effective results from your beta testing efforts.
+
+---
+
+Beta testing is done during both the proposal (pull request) and pre-release (nightly build) phases of development:
+
+## Testing Pull Requests
+
+Tester builds are automatically created for every [pull request](https://docs.github.com/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) (PR) that proposes a relevant change.
+
+The builds are updated if the author pushes changes to the PR.
+
+### Installation
+
+The tester build for a PR can be downloaded by following these instructions:
+
+1. Sign in to your [**GitHub**](https://github.com/) account.<br />
+   (GitHub only allows downloads of the tester builds when you are signed in.)
+1. Open the PR you are interested in.<br />
+   They are listed here:<br />
+   https://github.com/arduino/arduino-ide/pulls
+1. Click the "**Checks**" tab at the top of the PR's page.
+1. From the list on the left side of the page, click on "**Arduino IDE**".
+1. Scroll down to the "**Artifacts**" section of the page that opens.
+1. Click the download link for your operating system.<br />
+   **ⓘ** For example, if you are using Windows, click the "**Windows_X86-64_zip**" link.
+1. Wait for the download to finish.
+1. Extract or install the downloaded file as usual.
+
+![checks tab](assets/checks-tab.png)
+
+![tester build link](assets/tester-build-link.png)
+
+![tester build artifacts](assets/tester-build-artifacts.png)
+
+#### Notes for macOS
+
+Beginning in macOS 10.14.5, the software [must be notarized to run](https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution).
+
+For security reasons, signing and notarization are disabled when creating tester builds for pull requests from forks of this repository. This means that macOS will block you from running the tester builds for those PRs.
+
+Due to this limitation, Mac users have two options for testing PRs from forks:
```

Comment:
There is a third one, the user-friendly one:
 - Open `System Preferences` > `Security & Privacy` > `General` > Allow IDE2 to run in the `Allow apps downloaded...` section. You will be prompted for your password.

Reply:
Thanks @kittaakos! I think that is a much better approach.

I decided to replace the current "risky approach" instructions with the ones you suggested. Please let me know what you think:

https://github.com/arduino/arduino-ide/pull/1419/commits/eb6fe4f3b720af7acb86014c2a7acc7a2cc088bc

https://github.com/per1234/arduino-ide/blob/contributor-guide/docs/contributor-guide/beta-testing.md#the-risky-approach

Reply:
Perfect 👍 

I am resolving this thread.